Problems with building of pkcs15-tool.c

### Expected behaviour

I downloaded the current OpenSC master from the repository in order to install it on my system (debian 4.8.4-1).

### Actual behaviour

Alfter running ./bootstrap, ./configure and make the following error is thrown:
`pkcs15-tool.c: In function ‘list_data_objects’:
pkcs15-tool.c:533:5: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 2 has type ‘size_t’ [-Werror=format=]
     printf("  Size:%5lu", cinfo->data.len);
     ^
cc1: all warnings being treated as errors
Makefile:723: recipe for target 'pkcs15-tool.o' failed
make[3]: *** [pkcs15-tool.o] Error 1
make[3]: Leaving directory '/opt/OpenSC-master/src/tools'
Makefile:410: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/opt/OpenSC-master/src'
Makefile:538: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/opt/OpenSC-master'
Makefile:444: recipe for target 'all' failed`


### Steps to reproduce

I configured OpenSC with the following options:


`
OpenSC has been configured with the following options:


Version:                 0.16.0
Version fix:             0
Version revision:        0
Git revision:            No Git revision info available

Copyright:               OpenSC Project
Company:                 OpenSC Project
Company URL:             https://github.com/OpenSC
Comments:                Provided under the terms of the GNU Lesser General Public License (LGPLv2.1+).
Product name:            OpenSC smartcard framework
Product updates:         https://github.com/OpenSC/OpenSC/releases
Product URL:             https://github.com/OpenSC/OpenSC

User binaries:           /usr/local/openssl/bin
Configuration files:     /usr/local/openssl/etc
XSL stylesheets:         no

man support:             no
doc support:             no
thread locking support:  yes
zlib support:            yes
readline support:        no
OpenSSL support:         yes
PC/SC support:           yes
OpenCT support:          no
CT-API support:          no
minidriver support:      no
SM support:              yes
SM default module:       libsmm-local.so
DNIe UI support:         no
Debug file:              /tmp/opensc-debug.log

PC/SC default provider:  /usr/lib/libpcsclite.so
PKCS11 default provider: opensc-pkcs11.so

Host:                    i686-pc-linux-gnu
Compiler:                gcc
Preprocessor flags:      -DDEBUG -DPKCS11_THREAD_LOCKING
Compiler flags:          -fno-strict-aliasing -g -Werror -Wall -Wextra -Wno-unused-parameter -Werror=declaration-after-statement
Linker flags:            -g -Wl,-rpath,/usr/local/lib
Libraries:               -ldl 

READLINE_CFLAGS:         
READLINE_LIBS:           
ZLIB_CFLAGS:             
ZLIB_LIBS:               -lz
OPENSSL_CFLAGS:          -I/usr/local/include
OPENSSL_LIBS:            -L/usr/local/lib -lcrypto
OPENCT_CFLAGS:           
OPENCT_LIBS:             
PCSC_CFLAGS:             -pthread -I/usr/include/PCSC
`

The openssl directories are right.

What is the reason for this error?

--- a/src/tools/pkcs15-tool.c
+++ b/src/tools/pkcs15-tool.c
@@ -530,7 +530,7 @@ static int list_data_objects(void)
 					return 1;
 				}
 				sc_pkcs15_free_data_object(data_object);
-				printf("  Size:%5lu", cinfo->data.len);
+				printf("  Size:%5lu", (unsigned long) cinfo->data.len);
 			} else {
 				printf("  AuthID:%-3s", sc_pkcs15_print_id(&objs[i]->auth_id));
 			}

